astropy/utils/iers/tests/test_iers.py

```diff
@@ -302,11 +303,15 @@ def test_simple(self):
 
             # Look at times before and after the test file begins.  This forces a new download.
             assert np.allclose(dat.ut1_utc(Time(50000, format='mjd').jd).value, 0.1292905)
-            assert np.allclose(dat.ut1_utc(Time(60000, format='mjd').jd).value, -0.3)
+            # https://github.com/astropy/astropy/issues/9600
+            with pytest.raises(ValueError):
+                assert np.allclose(dat.ut1_utc(Time(60000, format='mjd').jd).value, -0.3)
```

Comment:
This doesn't seem right (to make it seem like the exception is wanted). I think an xfail on the test would make more sense here?